library/src/main/java/io/getstream/chat/android/client/controllers/ChannelControllerImpl.kt

```diff
@@ -35,7 +86,59 @@ internal class ChannelControllerImpl(
 
     override fun events(): ChatObservable {
         return client.events().filter { event ->
-            event.isFrom(cid)
+            when (event) {
+                is ChannelCreatedEvent -> event.cid == cid
```

Comment:
@tbarbugli or @thesyncim thoughts on this? it seems overly complicated to have to get the cid like this